docs/about/glossary.md

```diff
@@ -1,117 +1,116 @@
 # Glossary
 
-Adapted from [The World of Statecharts (Glossary)](https://statecharts.dev/glossary/).
+This glossary is a guide to the most common terms in statecharts and state machines.
 
-## Action
+## State machines
 
-An action is an [effect](../guides/effects.md) that is executed as a reaction to a state transition. Actions are fire-and-forget; that is, they are executed without needing to wait for a response.
+A state machine is a model showing how you transition from state to state in a process when events occur.
```

Comment:
```suggestion
A state machine is an application model that ensures it will only be in one state at a time.
```

I stumbled a bit on the description of a state machine. Not sure my suggestion is much better though 😆

Reply:
Discussed on Discord:

```suggestion
A state machine is a model that describes how the state of a process transitions to another state when an event occurs. State machines make building reliable software easier because they prevent impossible states and undesired transitions.
```